t/test-genome-DBs/homo_sapiens/funcgen/table.sql

```diff
@@ -407,15 +426,32 @@ CREATE TABLE `motif_feature` (
   `seq_region_start` int(10) unsigned NOT NULL,
   `seq_region_end` int(10) unsigned NOT NULL,
   `seq_region_strand` tinyint(1) NOT NULL,
-  `display_label` varchar(60) DEFAULT NULL,
   `score` double DEFAULT NULL,
-  `interdb_stable_id` mediumint(8) unsigned DEFAULT NULL,
+  `stable_id` varchar(18) DEFAULT NULL,
   PRIMARY KEY (`motif_feature_id`),
-  UNIQUE KEY `interdb_stable_id_idx` (`interdb_stable_id`),
+  UNIQUE KEY `unique_idx` (`binding_matrix_id`,`seq_region_id`,`seq_region_start`,`seq_region_strand`),
   KEY `seq_region_idx` (`seq_region_id`,`seq_region_start`),
   KEY `binding_matrix_idx` (`binding_matrix_id`)
 ) ENGINE=MyISAM AUTO_INCREMENT=1290348 DEFAULT CHARSET=latin1;
 
+CREATE TABLE `motif_feature_peak` (
+  `motif_feature_peak_id` int(11) NOT NULL AUTO_INCREMENT,
+  `motif_feature_id` int(11) NOT NULL,
+  `peak_id` int(11) NOT NULL,
+  PRIMARY KEY (`motif_feature_peak_id`),
+  UNIQUE KEY `motif_feature_peak_idx` (`motif_feature_id`,`peak_id`)
+) ENGINE=MyISAM DEFAULT CHARSET=latin1;
+
+CREATE TABLE `motif_feature_regulatory_feature` (
+  `motif_feature_regulatory_feature_id` int(11) NOT NULL AUTO_INCREMENT,
+  `motif_feature_id` int(11) NOT NULL,
+  `regulatory_feature_id` int(11) NOT NULL,
+  `epigenome_id` int(11),
+  `has_matching_Peak` tinyint(3) unsigned DEFAULT '0',
```

Comment:
0 is an interesting default. We've been trying to stamp them out of the core schema. Outside of the scope of this pull-request though.

Reply:
This table will probably be re-designed in the upcoming releases.